docs/configuration.asciidoc

```diff
@@ -360,4 +394,21 @@ You can use this setting to dynamically disable Elastic APM at runtime.
 |============
 | Default                          | Type                | Dynamic
 | `true` | Boolean | true
-|============
\ No newline at end of file
+|============
+
+[float]
+[[config-session-sample-rate]]
+==== `session_sample_rate` (added[0.9.0])
+
+By default, the agent will sample all signals generated by your application (e.g. spans, metrics, & logs).
+To reduce overhead and storage requirements, you can set the sample rate to a value between 0.0 and 1.0. When reduced below 1.0, data will be sampled per session.
+This is so context in a given session isn't lost.
+You can use this setting to dynamically disable Elastic APM at runtime.
+
```

Comment:
We have some good docs in the APM Guide that explain the sample rate in a bit more detail. Should we consider linking to them for more information? https://www.elastic.co/guide/en/apm/guide/current/sampling.html

Reply:
Thanks for letting me know! I wasn't aware of those and I generally think that the more info we provide the better! However in this case it seems like the docs' sampling rate approach is slightly different from what we do on mobile so adding that link here might cause some confusion. The approach in the docs mentions that a percentage of transactions will be discarded based on the sample rate, but we realized that for mobile it's better to drop all the signals/transactions from a session (or send them all) based on the sample rate so that we wouldn't get incomplete sessions later. Though I think we could at some point create or expand those docs to mention the "session sampling" approach too!

Reply:
Ahh, okay makes sense to me!